<?xml version="1.0"?>
<launch>
    <arg name="model_ckpt_file" default=""/>

    <rosparam command="load" file="$(find ur5_rl)/config/reach_goal_params.yaml" />
    <param name="n_episodes" value="10000"/>
    <param name="nenvs" value="4"/>
    <param name="n_steps_per_episode" value="250"/>
    <param name="learning_rate" value="0.00001"/>
    <param name="ckpt_freq" value="250"/>
    <param name="ckpt_file" value="$(arg model_ckpt_file)"/>
    <param name="num_runner_epochs" value="10"/>
    <param name="num_runner_minibatches" value="25"/>
    <param name="world_launch_file" value="$(find ur5_gazebo)/launch/ur5_cubes.launch"/>
    <node pkg="ur5_rl" name="ur5_gym" type="ppo_main_multi.py" output="screen" />

</launch>